sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts

```diff
@@ -238,7 +238,7 @@ export class EducationProgramOfferingInstitutionsController extends BaseControll
       locationId,
       programId,
       offeringId,
-      true,
+      { isEditOnly: true, isIncludeInActiveProgram: false },
```

Comment:
First `IsEditOnly` sounds like front end feature and second, why do we need it ?

like wise why do we need `isIncludeInActiveProgram ` ?

Reply:
If the intention is to validate the active program, I would recommend something like this. 

```ts
const offering = await this.programOfferingService.getProgramOffering(
      locationId,
      programId,
      offeringId,
    );
    if (!offering) {
      throw new UnprocessableEntityException(
        "Either offering for the program and location is not found or the offering is not in the appropriate status to be updated.",
      );
    }

    if (offering.educationProgram.isActive) {
      throw new UnprocessableEntityException(
        "Program is not active and the offering cannot be updated.",
      );
    }
```